addons/mass_mailing/wizard/mail_compose_message.py

```diff
@@ -78,7 +78,7 @@ def get_mail_values(self, res_ids):
                     'email': mail_to,
                 }
                 if mail_values.get('body_html') and mass_mail_layout:
-                    mail_values['body_html'] = mass_mail_layout._render({'body': mail_values['body_html']}, engine='ir.qweb', minimal_qcontext=True)
+                    mail_values['body_html'] = mass_mail_layout._render({'body': mail_values['body_html']}, engine='ir.qweb', minimal_qcontext=True).decode()
```

Comment:
Why this change ? Looking at number of rendering we do in various places, did we miss some encode/decode elsewhere ?

Reply:
See [the commit message](https://github.com/odoo/odoo/pull/68072/commits/d254075871ebf4e9a25d8fd1e3fe9d60d491a2d2).

Most such situations are implicitly decoded because they go through a to_text/ustr/whatever, this one very specifically is not.

Reply:
I have to say your message does not really helps dumb people like me understanding the real issue. What is this "Running with `-b`" ?

Reply:
> What is this "Running with -b" ?

`python -b <...>`

> I have to say your message does not really helps dumb people like me understanding the real issue.

```python
>>> rendered = b'<div>whee</div>'
>>> str(rendered)
"b'<div>whee</div>'"
>>> Markup(rendered)
Markup("b'<div>whee</div>'")
```